### ISSUE TITLE
I made a list of my zigbee devices. They are stored in TOTAL_LIST. I also made a list of the names in TOTAL_LIST_NAMES.

### ISSUE THREAD

**Issue by huk13** [sbormann/ioBroker.iqontrol#239]
**Describe the bug**  

**To Reproduce**  
2. Click on '...'

**Expected behavior**  

**Screenshots & Logfiles**  
If applicable, add screenshots and logfiles to help explain your problem.

**Versions:**  
 - Adapter version: <adapter-version>
 - JS-Controller version: <js-controller-version> 
 - Node version: <node-version> 
 - Operating system: <os-name>

**Additional context**

**Comment by huk13** [comment]
I made a list of my zigbee devices. They are stored in TOTAL_LIST. I also made a list of the names in TOTAL_LIST_NAMES. The last 4 names in the list of names are: Wz Süd Rollo, Wz TV LED1732G11, Wz Terrassen Rollo, Wz Terrassentür.
The datapoints are sorted according to: Wz Süd Rollo, Wz Terrassen Rollo, Wz Terrassentür, Wz TV LED1732G11.
I guess words with multiple capital letters (TV) are not sorted alphabetically in the LIST_NAMES but are in the TOTAL_LIST of datapoints
I use the latest version of iQontrol (2.00 with all revisions).

**Comment by sbormann** [comment]
Hi, sorry, ich verstehe den Fehler nicht. Kannst Du noch mal genauer beschreiben, was nicht stimmt?

**Comment by huk13** [comment]
Hallo,

Soweit ich die Listen verstehe sollten die Einträge in den Listen TOTAL_LIST (zigbee.0.xxxx) und TOTAL_LIST_NAMES eins zu eins (alphabetisch entsprechend der TOTAL_LIST_NAMES)  sortiert sein. Das heißt, daß zum Beispiel das 25. Element in TOTAL_LIST dem 25. Element aus der TOTAL_LIST_NAMES entsprechen sollte. Das funktioniert auch bis zu dem Element Wz TV LED1732G11. Dieses wird in der NAMES Liste nicht nach Alphabet sortiert Das TV Element sollte am Schluß stehen. Das entsprechende Element in der Datenpunkt Liste (TOTAL_LIST) steht an letzter Stelle. Wie immer, beide Listen sollten so sortiert sein, daß sich die Elementenreihenfolgen entsprechen. In der TOTAL_LIST_NAMES wird TV nicht wie Tv sortiert.

Ich hoffe, daß ich es jetzt etwas klarer gemacht habe.

Grüße HUK

**Comment by sbormann** [comment]
Die Sortierreihenfolge kann man ja angeben. Hast Du das auf "Name aufsteigend" gesetzt?

**Comment by sbormann** [comment]
Ich glaube, ich habe etwas gefunden, was das erklären könnte.
Aktualisier bitte mal auf die aktuelle Github-Version und schau, ob es damit wie gewünscht funktioniert.
Danke

**Comment by huk13** [comment]
Ja, Du hast es gefunden. Jetzt laufen die beiden Listen synchron. Du hast die alphabetische Sortierung so gewählt, daß TV nicht wie Tv sortiert wird sondern nach Tz.
Danke für die Mühe. IQontrol ist inzwischen so mächtig geworden, daß es für mich herausfordernd ist, immer mehr Dinge auszunutzen. Listen sind jetzt dran.
HUK
